ONSdigital/dp-api-router
# Added /health routing and migrated to dp-net context

### What

- Added image API proxy, with tests
- Migrated api-router and all dependencies to dp-net

### How to review

- Make sure code changes make sense
- Unit tests should pass

- You may run an integration test against imageAPI:
  - Run image api (`make debug` in image api)
  - Run dp-api-router (`make debug` in dp-api-router, with enabled auditing - optional)
  - (optional) Run auditing validation tool: dp-data-tools/check-audit-new
  - Make a request against image api through dp-api router: e.g.` GET http://localhost:23200/v1/images`

### Who can review

Anyone
